src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.List;
 import mate.academy.dao.MovieDao;
+import mate.academy.exception.DataProcessingException;
```

Comment:
It's better to use a more specific exception, like NoSuchElementException, instead of DataProcessingException.